### PR TITLE
0.41.0 release

### PR DESCRIPTION
**Pull request by natestemen**
## Description

Does what it says on the 🥫.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,34 @@
...
+
+#### 🧑🏽‍💻 Developer Improvements
+
+- remove failing test; simplify layerwise ZNE tests (#2545) [@natestemen]
+
+### 📞 Call for ideas
+
+We're currently looking into what features we could add to make Mitiq more **noise-aware**.
+If you have ideas and features requests in this area, do get in touch!
+Contact Nate at `nate@unitary.fund`.
```
Comment (cosenal) [inline CHANGELOG.md:32]: I would direct people to a github discussion instead.

Comment (Misty-W) [reply CHANGELOG.md:32]: +1 for directing to a github discussion. I also like the newsletter idea, we should add a call for ideas to the next edition.

Comment (natestemen) [reply CHANGELOG.md]: Good call. Replaced with a link to the discussion (https://github.com/unitaryfund/mitiq/pull/2557/commits/512529c3283a4a9a5f91c98e6ef18105224d61ef), and will add a note in the future newsletter.

**Review by Misty-W — APPROVED** [review]
LGTM, thanks @natestemen!

**Review by cosenal — APPROVED** [review]
✂️ it!

**Pull request merged**